### PR TITLE
Update jackson version

### PR DESCRIPTION
**Pull request by a1exsh**
Use the latest one.

### CODE REVIEW

**Comment by slitsche** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Comment by a1exsh** [comment]
It doesn't build:
```
[ERROR] Failed to execute goal on project pgobserver-gatherer: Could not resolve dependencies for project de.zalando.pgobserver:pgobserver-gatherer:jar:1.0.12: The following artifacts could not be resolved: com.fasterxml.jackson.core:jackson-core:jar:2.9.9.1, com.fasterxml.jackson.core:jackson-annotations:jar:2.9.9.1, com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.9.1: Could not find artifact com.fasterxml.jackson.core:jackson-core:jar:2.9.9.1 in maven (https://maven.zalando.net/content/groups/public)
```

It seems that only `jackson-databind` got a patched version released, but not the others...

File: gatherer/pom.xml [hunk 1]
```diff
@@ -96,7 +96,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${jackson.xml.version}</version>
+            <version>${jackson.xml.version}.2</version><!-- NB: 4th digit release! -->
```
Comment (a1exsh) [inline gatherer/pom.xml]: @slitsche this is quite obscure, but I don't have better ideas at the moment. :-/

**Comment by slitsche** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**